test.js

```diff
@@ -44,5 +44,6 @@ test(function (t) {
   t.equal(interval('00:00:00.100500').toISO(), 'P0Y0M0DT0H0M0.1005S')
   t.equal(interval('00:00:00.123456').toISO(), 'P0Y0M0DT0H0M0.123456S')
   t.equal(interval('-00:00:00.123456').toISO(), 'P0Y0M0DT0H0M-0.123456S')
+  t.equal(interval('-00:00:00.123456').toISOString(), 'P0Y0M0DT0H0M-0.123456S')
```

Comment:
Can you update the tests as well to flip toISO/toISOString? Otherwise LGTM.

Reply:
Done.